cmake/SetupCppCheck.cmake

```diff
@@ -6,19 +6,41 @@ if(NOT CPPCHECK_FOUND AND NOT CPPCHECK_EXECUTABLE)
 endif()
 
 if(CPPCHECK_EXECUTABLE)
+  # The reason for checking if the printed out file is empty is because
```

Comment:
But that's not why we redirect the output. We redirect the output because this way it's easier to spot the errors since they can be printed afterwards.

Reply:
Fair enough. Then I would just suggest rewording "the printed out file" -> "the output file `.cppcheck_output.txt`" to be explicit.

Reply:
Sure